src/pages/ProviderProfilePage/index.jsx

```diff
@@ -0,0 +1,162 @@
+import { useState, useEffect } from 'react';
+import ProfileCard from '../../components/user/ProviderProfilePageComponents/ProfileCard';
+import AboutSection from '../../components/user/ProviderProfilePageComponents/AboutSection';
+import ServicesSection from '../../components/user/ProviderProfilePageComponents/ServicesSection';
+import ReviewsSection from '../../components/user/ProviderProfilePageComponents/ReviewsSection';
+import ButtonComponent from '../../components/reUsableComponents/ButtonComponent';
+import ProfilePic from '../../components/user/ProviderProfilePageComponents/ProfilePic';
+import {
+    FaWrench,
+    FaMobileAlt,
+    FaWater,
+    FaVolumeUp,
+    FaLock,
+    FaBroom,
+    FaLightbulb,
+} from 'react-icons/fa';
+import MediaGrid from '../../components/user/ProviderProfilePageComponents/MediaGrid';
+
+const ProviderProfile = () => {
+    const [profileData, setProfileData] = useState(null);
+    const [aboutDescription, setAboutDescription] = useState('');
+    const [servicesData, setServicesData] = useState([]);
+    const [reviewsData, setReviewsData] = useState([]);
+    const [imagesData, setImagesData] = useState([]);
+    const [videosData, setVideosData] = useState([]);
+    const [loading, setLoading] = useState(true);
+    const [error, setError] = useState();
+
+    useEffect(() => {
+        const fetchData = async () => {
+            try {
+                // Simulate API call with a timeout
+                const data = await new Promise((resolve) => {
+                    setTimeout(() => {
+                        resolve({
+                            profileData: {
+                                distance: 3,
+                                rating: 4.2,
+                                reviews: 32,
+                                name: 'William S. Cunningham',
+                                occupation: 'Plumber',
+                                mediaCount: 21,
+                                servicesListed: 2,
+                                imageUrl: 'https://randomuser.me/api/portraits/men/75.jpg',
+                            },
+                            aboutDescription: `
+                Graphic Design is now a popular profession. It offers various opportunities in the fields of marketing, advertising, web design, and many more.
+                Whether you're working on digital media or print, understanding design principles is key. Beyond the aesthetic appeal, graphic designers must also
+                focus on delivering a message effectively. The profession is challenging but rewarding, as it allows individuals to use both creativity and strategy.
+              `,
+                            servicesData: [
```

Comment:
if data is big make it in a different component

Reply:
I’ve updated the code as suggested. I added separate hooks with the fetch data logic and imported useProviderProfileData from components/user/ProviderProfilePageComponents/hooks/useProviderProfileData